src/clj/rems/api/catalogue.clj

```diff
@@ -11,6 +11,8 @@
 (def GetCatalogueResponse
   [CatalogueItem])
 
+;; TODO resid is misleading: it's the internal id, not the string id
```

Comment:
This is also confusing in the database. It requires some thought to change as the migrations probably need fixing too.